app/services/create_account.rb

```diff
@@ -41,6 +42,10 @@ def create_redis_namespace
     CreateRedisNamespaceJob.perform_later(account)
   end
 
+  def create_default_admin_set
+    CreateDefaultAdminSetJob.perform_later(account)
```

Comment:
@atz @jcoyne @cbeer An admittedly janky workaround that abides by both of the above principles might be a job that intentionally fails until both fcrepo and solr are present, and then fires off. Too janky?